apps/web/app/(booking-page-wrapper)/layout.tsx

```diff
@@ -13,12 +13,7 @@ export default async function BookingPageWrapperLayout({ children }: { children:
   const ssr = await ssrInit(context);
 
   return (
-    <PageWrapper
-      isBookingPage={true}
-      requiresLicense={false}
-      nonce={nonce}
-      themeBasis={null}
```

Comment:
`themeBasis` is now computed directly by getThemeProviderProps which needs it.

Changed the approach as with App Router, it doesn;t see to be possible to pass it from server, plus it seems to be unnecessary overhead.